client/src/app/site/pages/meetings/pages/motions/services/export/motion-pdf.service/motion-pdf.service.ts

```diff
@@ -331,6 +331,15 @@ export class MotionPdfService {
         if (!infoToExport || infoToExport.includes(`referring_motions`)) {
             if (motion.referenced_in_motion_recommendation_extensions.length) {
                 const referringMotions = motion.referenced_in_motion_recommendation_extensions
+                    .sort((a, b) =>
+                        a.number
+                            ? b.number
+                                ? a.number.localeCompare(b.number)
+                                : -1
+                            : b.number
+                            ? 1
+                            : a.title.localeCompare(b.title)
+                    )
                     .map(motion => motion.getNumberOrTitle())
```

Comment:
Can't you just sort after this map? Then you don't have to differentiate whether the motions have numbers or not

Reply:
If I want motions without numbers to end up at the end of the list, I'll either have to do this or use strange string operations in order to guess whether what I'm getting is a number or a title

Reply:
ah, okay, did not know this was a requirement